src/common/playlists/PlayList.jsx

```diff
@@ -8,7 +10,11 @@ const PlayList = () => {
   if (plays.length === 0) {
     return (
         <div className="play-not-found">
-          <h2>Play Not Found</h2>
+          <ImageOops className="play-not-found-image" />
+          <p className='page-404-lead'>Play not found</p>
+          <p className='page-404-desc'>
+          Why don't you try anther play name or go back to <Link to="/" className='page-404-link'>home</Link>?
```

Comment:
@nirmalkc it would be better to link it to the `/plays` that is the playlist page.

Why don't you try anther play name or go back to the <Link to="/plays" className='page-404-link'>playlist</Link>?

Reply:
@atapas playlist is driven by `searchFilter` and rendering under `/plays` page, so "going back" does it make sense?  Ideally, the workflow should navigate the user to update the searchfilter or clear it. 
am I missing anything?

Reply:
Thanks for catching that! Yes my intention was not to go to the home page. We need to tackle the sentence here. Do you have a suggestion?

Reply:
i would suggest two part solution
1. Have the play search text box also be included in filter modal
2. on `404 play` page, upon clicking the link, it should open the modal

for the scope of this PR, I would suggest to take care of #2. And create an issue for #1.

Reply:
If we attempt # 1 as a future issue, I would even reduce the scope of this one. I would just say,

"Please readjust your search or filter to find plays."

Reply:
works for me. So link or buttons

Reply:
No link, no button for now. Just the text suggesting to adjust the search or filter criteria is what I am suggesting.